test/scripts/deploy_helper.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
I'm not a huge fan of adding another file with almost exactly the same name as an existing one